server/utils/db/db_utils.py

```
@@ -1276,6 +1328,22 @@ def initialize_tables():
 
+            # Migration: Add installation_id column to github_connected_repos for
+            # GitHub App-linked repos while preserving legacy OAuth rows as NULL.
+            try:
+                cursor.execute(
+                    "ALTER TABLE github_connected_repos ADD COLUMN IF NOT EXISTS installation_id BIGINT NULL;"
+                )
+                conn.commit()
+                logging.info(
+                    "Ensured installation_id column exists on github_connected_repos table."
+                )
+            except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)